Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -337,10 +340,10 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
         private async Task DoInitializeAsync(InitializeParams @params) {
             _analyzer = await CreateAnalyzer(@params.initializationOptions.interpreter);
 
-            _clientCaps = @params.capabilities;
-            _settings.SetCompletionTimeout(_clientCaps?.python?.completionsTimeout);
-            _traceLogging = _clientCaps?.python?.traceLogging ?? false;
-            _analyzer.EnableDiagnostics = _clientCaps?.python?.liveLinting ?? false;
+            _clientCaps = new Capabilities(@params.capabilities);
+            _settings.SetCompletionTimeout(_clientCaps.Python.CompletionsTimeout);
+            _traceLogging = _clientCaps.Python.TraceLogging;
+            _analyzer.EnableDiagnostics = _clientCaps.Python.LiveLinting;
```

Comment:
`LiveLinting` is currently false by default, since we have other messages behind that flag that we do not want to release. If unresolved imports are going there too, we might need an alternative filter somewhere in the analyzer to allow "good" diagnostics.

The `LiveLinting` flag is meant to enable experimental stuff, so maybe internally we can change it to `EnableExperimentalDiagnostics` and filter those somehow?

Reply:
Yeah, unresolved imports should be default on (as warning). We can expose controls for severity like VS does so user can suppress warnings if they chose to.

Reply:
Another alternative is create new structure with flags for assorted diagnostic controls and I can expose it in the VSC settings. We can then keep expanding it as needed

Reply:
`_analyzer.EnableDiagnostics` doesn't affect unresolved imports, and yes, they are reported as warnings.
I like the idea of having granulated diagnostics controls on the client side, that we can switch from `false` to `true` as needed.